packages/puppeteer-core/src/node/ChromeLauncher.ts

```diff
@@ -174,9 +175,12 @@ export class ChromeLauncher extends ProductLauncher {
       'OptimizationHints',
       // https://crbug.com/1492053
       'ProcessPerSiteUpToMainFrameThreshold',
-    ];
+    ].concat(getFeatures('--disable-features', options.args));
```

Comment:
nit: slight preference for the destucturing

Reply:
What do you mean? `, {args}`?

From `export function getFeatures(flag: string, options: string[] = []): string[] {`

To `export function getFeatures(flag: string, options: {options: string[]} = {options: []}): string[] {`

?

Reply:
Ah, I guess you mean the spread operator

Reply:
See the suggestion

Reply:
Yeah, sorry :)